common/web/keyboard-processor/src/keyboards/activeLayout.ts

```diff
@@ -550,6 +555,8 @@ namespace com.keyman.keyboards {
             // Results in a more optimized distribution.
             if(text.Codes.isKnownOSKModifierKey(key.baseKeyID)) {
               return;
+            } else if(key.isPadding) { // to the user, blank / padding keys do not exist.
```

Comment:
```suggestion
            } else if(key.isPadding) { // to the user, padding keys do not exist.
```

Per comment in `isPadding()` you are not testing for blank keys

Reply:
Ah, right, blank vs hidden.